docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: '2'
+services:
+  db:
+    image: postgres
```

Comment:
Good catch. I left the defaults from the postgres image and they are hard-coded into the settings.docker file